DataFormats/MuonReco/src/Muon.cc

```diff
@@ -147,6 +166,23 @@ unsigned int Muon::stationMask( ArbitrationType type ) const
          continue;
       }
 
+      if(type == ME0HitAndTrackArbitration) {
+      	 if(chamberMatch->me0Matches.empty()) continue;
+
+
+         for( std::vector<MuonSegmentMatch>::const_iterator me0Match = chamberMatch->me0Matches.begin();
+      	      me0Match != chamberMatch->me0Matches.end(); me0Match++ )
+      	   {
+      	     //Not sure if mask is correct..
+      	     curMask = 1<<( (chamberMatch->station()-1)+4*(chamberMatch->detector()-1) );
```

Comment:
... actually, 5 is not that good. 
If you keep me0Matches separately, then this can be simply 1; if the me0 segments are merged with regular (DT/CSC) segment matches, then this can be 8 (DTs go from 0 to 3; CSC from 4 to 7).